README.md

```diff
@@ -0,0 +1,121 @@
+# Sanity plugin duplicate
+Adds (usable) duplicate document action to Sanity.
+
+## Installation
+
+```
+> cd admin
+> yarn add @kaliber/sanity-plugin-duplicate
+```
+
+_`admin/sanity.json`_
+
+```json
+{
+  "plugins": [
+    "@kaliber/sanity-plugin-duplicate",
+    ...
+  ],
+  ...
+}
```

Comment:
Volgens mij is dit stukje niet nodig.

Reply:
Deze is nodig om de config aan te maken voor het mulitlingual. Dit wordt ook gedaan in sanit-preview, dus heb het hier ook toegeveogd